stocksite/stocks/views/buyView.py

```diff
@@ -12,13 +12,13 @@ class BuyView(APIView):
     """
     def post(self, request):
         # Get request data
-        username = request.data.get("username")
-        stockSymbol = request.data.get("stockSymbol")
-        amount = float(request.data.get("amount"))
+        username        = request.data.get("username")
+        stockSymbol     = request.data.get("stockSymbol")
+        transactionNum  = request.data.get("transactionNumber")
```

Comment:
Curious what the front-end would pass in for transaction number for an action on the website?

is it possible we can pass in something their to save a query later?

Reply:
Good point, we save big query time if we pass the transactNum through the view.

I was thinking that since the transactNums are really only a requirement because of the workload generator and the logfiles, we could add an if statement that checks if its passed through with the command or not. If its not we can set it to an Auto(Incrementing)Field or just set it to 0 or something

I'm not sure how things affect the front-end though

Reply:
ah yah good point! I remember there being a solid reason. 

I'll just set it too 0 if its not being passed in.